test/list.spec.js

```diff
@@ -265,6 +265,27 @@ describe('Linked List', function() {
             arrayedLists.forEach(function(d, i) {
                 expect(d).to.equal(newList.findAt(i).getData());
             });
+
+            //this list is empty, second is not
+            list.clear();
+            newList = list.concat(secondList);
+            expect(newList.getSize(), 5);
+            expect(newList.getHeadNode()).to.not.equal(null);
+            expect(newList.getTailNode()).to.not.equal(null);
```

Comment:
We have a lot expectations in this test...and we are testing different cases, so maybe we should split these out to their own tests.  Maybe something like...

```
it('when empty concats a populated list', () => {
    // set up and expectations here
});
```